yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -1032,52 +1091,121 @@ Internet-Draft                 YANG Semver                  October 2023
         Relating to IETF Documents
         (http://trustee.ietf.org/license-info).
 
+        The key words 'MUST', 'MUST NOT', 'REQUIRED', 'SHALL', 'SHALL
+        NOT', 'SHOULD', 'SHOULD NOT', 'RECOMMENDED', 'NOT RECOMMENDED',
+        'MAY', and 'OPTIONAL' in this document are to be interpreted as
+        described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
+        they appear in all capitals, as shown here.
+
         This version of this YANG module is part of RFC XXXX; see
         the RFC itself for full legal notices.";
 
      // RFC Ed.: update the date below with the date of RFC publication
      // and remove this note.
      // RFC Ed.: replace XXXX with actual RFC number and remove this
      // note.
-     // RFC Ed. update the rev:revision-label to "1.0.0".
+     // RFC Ed. update the ys:version to "1.0.0".
 
-     revision 2023-01-17 {
-       rev:label "1.0.0-draft-ietf-netmod-yang-semver-10";
+     revision 2024-01-22 {
+       ys:version "1.0.0-draft-ietf-netmod-yang-semver-13";
        description
          "Initial revision";
        reference
          "RFC XXXX: YANG Semantic Versioning.";
      }
 
+
+
+
+Clarke, et al.           Expires 24 August 2024                [Page 20]
+
+Internet-Draft                 YANG Semver                 February 2024
+
+
      /*
-      * Identities
+      * Extensions
       */
 
-     identity yang-semver {
-       base rev:revision-label-scheme-base;
+     extension version {
+       argument yang-semantic-version;
        description
-         "The revision-label scheme corresponds to the YANG Semver
-          scheme which is defined by the pattern in the 'version'
+         "The version can be used to provide an additional
+          identifier associated with a module or submodule
+          revision.
+
+          The format of the revision label argument MUST conform to the
```

Comment:
version identifier instead of revision label?

Reply:
Fixed.